pkg/aws/ec2/ec2_test.go

```diff
@@ -26,92 +28,6 @@ var (
 	logger = slog.New(slog.NewTextHandler(os.Stdout, nil))
 )
 
-func TestCollector_ListOnDemandPrices(t *testing.T) {
```

Comment:
I just moved it to the `pricing_map_test.go` file instead. That's a more appropriate place for it, since the function it's testing lives in `pricing_map.go`. The test is now called `Test_ListOnDemandPrices`, but has the exact same content.